test/automated-suite.js

```diff
@@ -0,0 +1,151 @@
+/*
```

Comment:
This one is tidier than sw-toolbox.

It uses the which node module so should work cross platform. It also only runs ff beta on travis because the file path is too specific to be useful anywhere else.